src/components/swapv2/PairSuggestion/SearchInput.tsx

```diff
@@ -109,7 +109,13 @@ export default forwardRef<HTMLInputElement, Props>(function SearchComponent(
         ref={ref}
         hasBorder={hasBorder}
         onBlur={onBlurInput}
-        onClick={showListViewWithTracking}
+        onFocus={
+          disabled
+            ? () => {
+                //
```

Comment:
Use `undefined` instead. Don't code like this.

Reply:
done